test/validator.h

```diff
@@ -444,5 +452,27 @@ void testValidate(
   }
 }
 
+// The variant with automatically inferred aten outputs
```

Comment:
Please mention that the given fusion is assumed to contain only supported expressions for ExpressionEvaluator.